_source/organisms/modal/Modal.scss

```diff
@@ -12,7 +12,7 @@
   align-items: center;
   transition: width 0s 0.2s linear;
 
-  &:before {
+  &::before {
```

Comment:
Consistency? :D U know that this is not wrong, tho?

Reply:
It is consistent, I'm usually using `::` for pseudo-elements.